examples/lighting-app/lighting-common/lighting-app.matter

```diff
@@ -1215,6 +1215,8 @@ server cluster Switch = 59 {
     INT8U totalNumberOfPressesCounted = 1;
   }
 
+  readonly attribute int8u numberOfPositions = 0;
```

Comment:
@AlexPygin This is a generated file. You'll want to modify the source file for this, which is `examples/lighting-app/lighting-common/lighting-app.zap`.

That said, whether to adjust this cluster's enabled attributes or whether to get rid of it altogether is in fact not very clear.  See https://github.com/project-chip/connectedhomeip/issues/24886

Reply:
I realized that because of the checks and added the proper changes in another commit.

I see. What should I do here then? Leave this open for now or delete the PR?

Reply:
I do not see any errors either, however at the same time the spec for switch says Number of positions and current position is mandatory ... maybe linter is not complete?

Reply:
When dumping the XML rules however I do see:

```
...
    - AttributeRequirement(code=0, name='NumberOfPositions', filter_cluster=59)
    - AttributeRequirement(code=1, name='CurrentPosition', filter_cluster=59)
...
```

So it does look like the xml-loaded rules should validate that this attribute is required and there.

Reply:
ah .. this is `light` not `light-switch`!

Reply:
@bzbarsky-apple : try:

```
./scripts/idl_lint.py --log-level warn examples/lighting-app/lighting-common/lighting-app.matter
```

it says:

```
2023-02-10 14:04:16 ERROR   ERROR: Required attributes: EP0:Switch does not expose NumberOfPositions(0) attribute at examples/lighting-app/lighting-common/lighting-app.matter:1949:3
2023-02-10 14:04:16 ERROR   ERROR: Required attributes: EP0:Switch does not expose CurrentPosition(1) attribute at examples/lighting-app/lighting-common/lighting-app.matter:1949:3
2023-02-10 14:04:16 ERROR   Found 2 lint errors
```

Reply:
Sorry, had wrong command pasted in original comment